README1.md

```diff
@@ -0,0 +1,13 @@
+in order to run the program on linux, instalation of vagrant and virtualbox is requierd installing vagrant:
+
+sudo apt update sudo apt install virtualbox
+
+curl -O https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_x86_64.deb sudo apt install ./vagrant_2.2.9_x86_64.deb
+
+after the instalation a directory must be created and the vagrant file provided in github must be copied there using the terminal
+
+mkdir vm sudo mv Vagrantfile vm
```

Comment:
Why not just run vagrant from the directory where Vagrantfile already is? There's no need to move it to another directory